README.md

```diff
@@ -238,5 +238,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Code-Server](/software/code-server)
 ### Grafana
 * [Grafana](/software/grafana)
+### Gitea
+* [Gitea](/software/gitea)
```

Comment:
Alphabetical order would put this above Grafana

Reply:
Whoops, sry. I'll correct that within the next 24h

Reply:
Any update @DatMayo?